### DISCUSSION TITLE
Would it be possible to unload beatmap covers less strictly?

### DISCUSSION THREAD

**Discussion by Phippe** [ppy/osu#26243] [Q&A] [answered]
I tried lazer again and one thing that stood out to me was the instant unloading of beatmap covers as soon as they're out of sight, which I assume this is done to improve performance. To me, this makes the game feel somewhat slow. If you scroll down in a browser and back up, you don't need to reload the images, they're still there. In lazer, you have to wait for the images to load again every time, even though they were there just a moment ago. So, would it be possible to keep the images in memory for longer and make the unloading more lenient?

**Comment by peppy** [comment, marked as answer]
Should be improved with #26385.
